modules/electrophysiology_browser/php/sessions.class.inc

```diff
@@ -53,6 +53,32 @@ class Sessions extends \NDB_Page
             ) && $user->hasProject($this->timepoint->getProject()->getId()));
     }
 
+    /**
+     * Determine whether the user has permission to view the derivatives
+     *
+     * @param \User $user The user whose access is being checked
+     *
+     * @return bool whether the user can view the derivatives
+     */
+    function _hasViewDerivPerms(\User $user) : bool
+    {
+        return ($user->hasPermission('electrophysiology_browser_view_annotations'));
```

Comment:
I think you're missing the superuser case :)

Reply:
I think the superuser case is handled in the `hasPermission` method at [this line](https://github.com/aces/Loris/blob/2fceb83b8f11bc2b315e279d260730e3d11a40a5/php/libraries/UserPermissions.class.inc#L110)

Reply:
True! My mistake, it's all good then.